docs/content/api.md

```diff
@@ -60,7 +60,9 @@ function GravityBasic() {
     infinite: true,
   });
 
-  return <div className="mover" {...props} />;
+  return (
+    <div className="live-preview__mover live-preview__mover--lg" {...props} />
```

Comment:
You may see a lot of changes like this in the diff. It's just migrating all our live code samples to use the BEM-style classes defined in `global-style.js`.